tests/custom_lookups/tests.py

```diff
@@ -12,6 +13,17 @@
 from .models import Author, MySQLUnixTimestamp
 
 
+@contextlib.contextmanager
+def register_lookup(model, *lookups):
```

Comment:
yes, I also renamed it another patch in my comment.
I will postpone it while we're discussing the issue above